sql/moz-fx-data-shared-prod/telemetry_derived/adm_engagements_daily_v1/metadata.yaml

```diff
@@ -1,6 +1,8 @@
 friendly_name: adMarketplace Daily Engagements
 description: |-
-  Click Volume, partitioned by day with the following facets:
+  Historical data on click volume through August 2021.
```

Comment:
I've removed the scheduling metadata for this query, and the DAG is now removed in this PR. But the query remains here for reference and the table continues to exist.